tests/hpc/gnu_compilers_hpc.pm

```diff
@@ -17,17 +17,15 @@ use Utils::Logging qw(export_logs);
 sub run ($self) {
     select_serial_terminal();
     my $version = get_required_var('GNU_COMPILERS_HPC_VERSION');
-    zypper_call("in gnu$version-compilers-hpc");
+    zypper_call("in gnu$version-compilers-hpc gnu$version-compilers-hpc-devel");
```

Comment:
why did you ad the devel? Is this needed?

Reply:
For each gnu$version-compilers-hpc a devel package should be updated too. Both packages should be tested according to https://progress.opensuse.org/issues/130844

Reply:
so the test here is to make sure the package is installable? Or is the devel rpm needed for rest of the things to work? 
In other words: will 
`    assert_script_run 'module load gnu';
    assert_script_run qq{env |grep "PATH=/usr/lib/hpc/compiler/gnu/$version/bin"};
    assert_script_run qq{gcc --version | grep $version};
`
fail if you don't add the devel package?

Reply:
And this is also what triggers the postscript of the spec which the fix is it for

Reply:
No it doesnt affect `module load gnu`. It is just to test installation of the package

Reply:
OK thanks for clarification; so please remove that change from the current git commit. Make the git commits atomic please; so add the devel in the next git commit with correct git commit message please.

Reply:
But checking the https://progress.opensuse.org/issues/130844 i need to add checks which i left out in first place due to the bug. thanks for making me review this. So finally answer is that it is/will be needed for other things to work

Reply:
> But checking the https://progress.opensuse.org/issues/130844 i need to add checks which i left out in first place due to the bug. thanks for making me review this. So finally answer is that it is/will be needed for other things to work

still, the git commits must be atomic :-)